src/appComponents/common/ApiJsonFactory.js

```diff
@@ -1,29 +1,46 @@
 /**
  * Created by championswimmer on 29/5/15.
  */
+(function () {
 
-var commonFactories = angular.module('openevent');
+"use strict";
+const commonFactories = angular.module("openevent");
+
+commonFactories.factory("ApiJsonFactory", ["$q", "$http", function ($q, $http) {
+
+const config = "" ;
+const baseUrl = `(config.use_testApi ? "testapi/" : config.apiBaseGetUrl) event/ ${config.eventId}`;
 
-commonFactories.factory('ApiJsonFactory', ['$q', '$http', function ($q, $http) {
-var baseUrl = (config.use_testApi?'testapi/':config.apiBaseGetUrl) +
-    'event/' + config.eventId;
         return {
             getJson: function ($apiEndpoint) {
-                let endpoint;
-                if (($apiEndpoint === 'event') && (!config.use_testApi)) {
-                    endpoint = baseUrl;
-                } else {
-                    endpoint = baseUrl + '/' + $apiEndpoint;
-                }
+                let endpoint = "";
+
+              if ($apiEndpoint === "event" && !config.use_testApi) {
+                 
+                  endpoint = baseUrl;
                 
-                var deferred = $q.defer(),
-                    httpPromise = $http.get(endpoint);
-                httpPromise.then(function (response) {
-                    deferred.resolve(response);
-                }, function (error) {
-                    console.error(error);
-                });
+              } 
+              else {
+                 
+                  endpoint = baseUrl + "/" + $apiEndpoint;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 18 space characters but found 17.](https://www.codacy.com/app/triplez/open-event-webapp/file/2433032601/issues/source?bid=3233857&fileBranchId=3238420#l25)